src/native.rs

```diff
@@ -196,6 +196,13 @@ impl HasContext for Context {
         Ok(NativeTexture(non_zero_gl_name(name)))
     }
 
+    unsafe fn create_textures(&self, target: u32) -> Result<Self::Texture, String> {
```

Comment:
The plural "textures" might be a bit confusing because we already have `create_textures` - maybe we could name this `create_texture_at_target` or similar?

Reply:
confusing names is the true spirit of opengl `(glEnableVertexAttribArray vs glEnableVertexArrayAttrib)` <3
I wanted to keep the names as is, so that people who are familiar with normal opengl can recognize them. any name is fine with me, if you are sure with `create_texture_at_target`, then, i will do that. the users will have to browse the source to make sure they are using the right function.